### PR TITLE
Improvement of favorite button re-render delay.

### PR DESCRIPTION
**Pull request by IzumiSy**
Decreases re-render delay of favorite button after clicking it. `dispatch` does not wait for API response anymore.

### CODE REVIEW

**Comment by k0kubun** [comment]
I think we should render it in both places in case of favorite request failure.

File: src/actions/tweets.js [hunk 1]
```diff
@@ -44,15 +44,17 @@ export const clearAndSetTweets = (tweets, account, tab) => {
...
-    const callback = (updatedTweet) => {
-      dispatch(addTweetToTab(updatedTweet, account, tab));
-    }
 
     if (tweet.favorited) {
-      client.unfavoriteStatus(tweet.id_str, callback);
+      client.unfavoriteStatus(tweet.id_str, (()=>{}));
     } else {
-      client.favoriteStatus(tweet.id_str, callback);
+      client.favoriteStatus(tweet.id_str, (()=>{}));
```
Comment (k0kubun) [inline src/actions/tweets.js]: I mean that we should keep`callback` in addition to `updateTweet`.

**Comment by IzumiSy** [comment]
Updated

**Comment by k0kubun** [comment]
:+1:

**Pull request merged**